sdk/src/types/block/output/feature/mod.rs

```diff
@@ -67,6 +71,17 @@ impl Ord for Feature {
         self.kind().cmp(&other.kind())
     }
 }
+impl core::hash::Hash for Feature {
+    fn hash<H: core::hash::Hasher>(&self, state: &mut H) {
+        self.kind().hash(state);
```

Comment:
Are we sure we want the feature hash to just be the kind? that would make 2 different SenderFeatures have the same hash

Reply:
just noticed that the Eq is also just kind. for our code it works, but what if someone gets outputs and compares it with their feature, youd just expect == to work.